AltStore/View Components/AppRowView.swift

```diff
@@ -20,7 +20,7 @@ struct AppRowView: View {
     
     var body: some View {
         HStack(alignment: .center, spacing: 12) {
-            AppIconView(iconUrl: storeApp?.iconURL)
+            AppIconView(iconUrl: storeApp?.iconURL, isSideStore: storeApp?.bundleIdentifier == Bundle.Info.appbundleIdentifier)
```

Comment:
The boolean should probably be an extension on the `StoreApp` class to prevent a lot of redundant code.